src/Schema/Directives/CountDirective.php

```diff
@@ -39,6 +39,18 @@ public static function definition(): string
   Apply scopes to the underlying query.
   """
   scopes: [String!]
+
+  """
+  The field to count.
+  counts only a specific field
+  """
+  field: String
```

Comment:
Sure, let me put together a few scenarios:
let's say we have 100 Projects with different states (the state column allows null values):

- Completed
- Draft
- Cancelled
- In Progress
- On Hold

Let's assume 5 of them have a null state.

| What we want  | Eloquent | lighthouse directive | Result |
| ------------- | ------------- | ------------- | ------------- |
| Count every row  | Project::count()  | @count(model: "Project") | 100 |
| Count every row excluding where state is null  | Project::count('state')  | @count(model: "Project", field: "state", distinct: true) | 95|
| Count every different state  | Project::distinct()->count('state')  | @count(model: "Project", field: "state", distinct: true) | 5|

Reply:
Great example. A test case for the effect of passing only `field` without `distinct` would be nice, asserting that it is indeed different from not passing `field`.

When looking at the signature of `count()`, I see that it can take more then one column. I don't know if it is a common scenario, but we can easily accomodate for it.

```suggestion
  """
  Count only rows where the given columns are non-null.
  `*` counts every row.
  """
  columns: [String!]! = ['*']
```